### PR TITLE
lchage: add page

### PR DESCRIPTION
**Pull request by shellheim**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/lchage.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# lchage
+
+> Display or change user password policy.
+> More information: <https://manned.org/lchage>.
+
+- Disable password expiration for the user:
+
+`sudo lchage --date=-1 {{username}}`
```
Comment (acuteenvy) [inline pages/linux/lchage.md:8]:
```suggestion
`sudo lchage --date -1 {{username}}`
```

File: pages/linux/lchage.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- List password policy for the user:
+
+`sudo lchage --list {{username}}`
+
+- Require password change for a user, certain days after the last password change:
+
+`sudo lchage --maxdays={{number_of_days}} {{username}}`
```
Comment (acuteenvy) [inline pages/linux/lchage.md:16]:
```suggestion
- Require password change for the user a certain number of days after the last password change:

`sudo lchage --maxdays {{number_of_days}} {{username}}`
```

File: pages/linux/lchage.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Start warning the user certain days before the password expires:
+
+`sudo lchage --warndays={{number_of_days}} {{username}}`
```
Comment (acuteenvy) [inline pages/linux/lchage.md:20]:
```suggestion
- Start warning the user a certain number of days before the password expires:

`sudo lchage --warndays {{number_of_days}} {{username}}`
```

File: pages/linux/lchage.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# lchage
+
+> Display or change user password policy.
+> More information: <https://manned.org/lchage>.
+
+- Disable password expiration for the user:
+
+`sudo lchage --date=-1 {{username}}`
+
+- List password policy for the user:
```
Comment (acuteenvy) [inline pages/linux/lchage.md:10]:
```suggestion
- Display the password policy for the user:
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**